fuzzbench/test_e2e_run.py

```diff
@@ -26,6 +28,13 @@
 class TestEndToEndRunResults:
     """Checks the result of a test experiment run."""
 
+    def test_jobs_dependency(self):
+        """Tests that jobs dependency preserves during working."""
+        jobs = Job.fetch_many(['base-image', 'base-runner', 'base-builder'],
+                              connection=redis.Redis(host='queue-server'))
+        assert jobs[0].ended_at <= jobs[1].started_at
```

Comment:
can we refer to jobs with their name (eg jobs['base-image'].ended_at <= jobs['base_runer'].started_at .. to make this more readable?

Reply:
Good suggestion!